go/util/receipts/receipts.go

```diff
@@ -0,0 +1,130 @@
+// Copyright 2016 Attic Labs, Inc. All rights reserved.
+// Licensed under the Apache License, version 2.0:
+// http://www.apache.org/licenses/LICENSE-2.0
+
+package receipts
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+	"fmt"
+	"net/url"
+	"time"
+
+	"github.com/attic-labs/noms/go/d"
+	"github.com/attic-labs/noms/go/hash"
+	"golang.org/x/crypto/nacl/secretbox"
+)
+
+// Data stores parsed receipt data.
+type Data struct {
+	Database string
+	Date     time.Time
+}
+
+// KeySize is the size in bytes of receipt keys.
+const KeySize = 32 // secretbox
+
+// Key is used to encrypt receipt data.
+type Key [KeySize]byte
+
+// Don't use a nonce when sealing/opening secretbox because these receipts need
+// to be used across sessions.
+var emptyNonce [24]byte
```

Comment:
This is surprising. I would have expected us to use random nonces... and to do that instead of the salt that you have. Were you following some specific instructions to do it this way?